### PR TITLE
aded the azure all billing profiles

### PR DESCRIPTION
**Pull request by athiruma**
1. Added the cost usage of two billing profiles

### CODE REVIEW

File: cloud_governance/policy/azure/cost_billing_reports.py [hunk 1]
```diff
@@ -34,21 +39,38 @@ def get_common_data(self):
...
+        common_data["AccountId"] = subscription_id
+        if account_name:
+            common_data['Account'] = account_name
+        if cost_center > 0:
+            common_data['CostCenter'] = cost_center
+        if subscription_id:
+            cost_center, allocated_budget, years = self.update_to_gsheet.get_cost_center_budget_details(account_id=subscription_id, dir_path='/tmp')
+            if cost_center:
+                common_data['CostCenter'] = int(cost_center)
+                common_data['Owner'] = 'Shai'
```
Comment (ebattat) [inline cloud_governance/policy/azure/cost_billing_reports.py:69]: Can you make 'Shai' environment variable / constant ?

File: cloud_governance/policy/azure/cost_billing_reports.py [hunk 2]
```diff
@@ -92,21 +126,50 @@ def collect_and_upload_cost_data(self):
...
-        uploaded_cost_data = list(cost_billing_data.values())
-        self.elastic_upload.es_upload_data(items=uploaded_cost_data, set_index='index_id')
-        upload_data = {
-            'cloud_alias_name': f'{self.azure_operations.account_name}-azure',
-            'cloud_name': 'AZURE',
-            'cloud_data': uploaded_cost_data
-        }
-        self.update_to_gsheet.update_data(cloud_data=upload_data)
-        return uploaded_cost_data
+        if not self.__total_account:
```
Comment (ebattat) [inline cloud_governance/policy/azure/cost_billing_reports.py:130]: Too long method, can you split it ?

File: cloud_governance/common/clouds/azure/cost_management/cost_management_operations.py [hunk 3]
```diff
@@ -15,54 +16,64 @@ def __init__(self):
...
+                if data_date in row_data:
+                    if row_data[data_date][2] == 'Actual' and data[2] == 'Forecast':
+                        row_data[data_date][2] = data[2]
+                    row_data[data_date][0] += data[0]
+                else:
+                    row_data[data_date] = data
+            result['rows'] = list(row_data.values())
+            return result
+        except Exception as err:
+            time.sleep(60)
```
Comment (ebattat) [inline cloud_governance/common/clouds/azure/cost_management/cost_management_operations.py:77]: Why we need this sleep after error ?
Can you put 60 as  constant ?

File: cloud_governance/policy/azure/cost_billing_reports.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/policy/azure/cost_billing_reports.py]: Added const varaibles

File: cloud_governance/common/clouds/azure/cost_management/cost_management_operations.py (hunk 3 above)
Comment (athiruma) [reply cloud_governance/common/clouds/azure/cost_management/cost_management_operations.py:77]: When we continuously start fetching, will get the limit exceeded. So we need to wait sometime.
ok will make constant

Comment (ebattat) [reply cloud_governance/common/clouds/azure/cost_management/cost_management_operations.py:77]: OK, so just make it constant SLEEP_TIME

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**